### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by witty123**
WHAT : 
Added sass framework in accordance with itcss for styling.
WHY:
For adding styling to dashboard

### CODE REVIEW

File: app/components/activity-chart/module.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+'use strict';
```
Comment (pranjalk) [inline app/components/activity-chart/module.js:1]: why strict???

File: app/components/activity-chart/service.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  service('activitySearch', ['$http', function($http){
+  return ({
+    getActivity: function(user, access_token){
+      return ($http.get("https://api.github.com/users/" + user +  "/events?page=1&per_page=300&access_token=" + access_token));
+    },
+    getSearch: function () {
+      return ($http.get("/api"));
+    }
+  });
+  }]);
```
Comment (pranjalk) [inline app/components/activity-chart/service.js]: add new line at end

File: app/components/activity-chart/template.html [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<div class="panel">
+    <span class = "small-box">Your activity</span>
+    <nvd3 options="options" data="data"></nvd3>
+</div>
```
Comment (pranjalk) [inline app/components/activity-chart/template.html]: add new line at end

File: app/components/language-chart/component.js [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+                    {
+                      $scope.data[lang].y+=1;
+                    }
+                  }
+                } 
+              } 
+          });
+        });
+    }
+]});
```
Comment (pranjalk) [inline app/components/language-chart/component.js]: add new line at end

File: app/components/language-chart/template.html [hunk 5]
```diff
@@ -0,0 +1,4 @@
+<div class="panel">
+    <span class = "small-box">Languages</span>
+    <nvd3 options="options" data="data"></nvd3>
+</div>
```
Comment (pranjalk) [inline app/components/language-chart/template.html]: new line at end

File: app/components/profile-chart/component.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+    controller: ['$scope', '$http', 'profileSearch',
+     function ProfileChartController($scope, $http, profileSearch) {
+      $scope.userData = {};
+      var userDataPromise = profileSearch.getSearch();
+      userDataPromise.then(
+        function (payLoad) {
+          $scope.userData = payLoad.data;
+      });
+  }]
+});
```
Comment (pranjalk) [inline app/components/profile-chart/component.js]: new line at end

File: app/components/profile-chart/service.js [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+
+angular.
+  module('profileChart').
+  service('profileSearch', ['$http', function($http){
+    return {
+    getSearch: function () {
+        return ($http.get("/api"));
+      }
+    };
+  }]);
```
Comment (pranjalk) [inline app/components/profile-chart/service.js]: new line at end

File: app/components/profile-chart/template.html [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+    <img class = "big-img" ng-src="{{userData._json.avatar_url}}">
+    </div>
+	<div>
+	    <div class="panel-heading">
+			<span class = "small-box">User: {{userData.username}} </span>
+			<span class = "small-box" >Followers: {{userData._json.followers}} </span>
+			<span class= "small-box" >Following: {{userData._json.following}} </span>
+		</div>
+	</div>
+</div>
```
Comment (pranjalk) [inline app/components/profile-chart/template.html]: new line at end

**Comment by pranjalk** [comment]
@witty123 : add new line in the end of all lines

File: app/components/activity-chart/module.js (hunk 1 above)
Comment (witty123) [reply app/components/activity-chart/module.js:1]: It will give me error if i have any undefined variables in this file.

**Comment by witty123** [comment]
@pranjalk Changes done.
@gautamvij Request for peer review.

File: app/components/language-chart/component.js [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+        {
+            key: "Others",
+            y:0
+        }
+      ];
+
+
+      var userDataPromise = languageSearch.getSearch();
+      userDataPromise.then( 
+        function(payLoad){
```
Comment (shreedharshetty) [inline app/components/language-chart/component.js]: name your variables based on the context and make sure its noun

**Pull request merged**